src/OpenTelemetry.Instrumentation.GrpcNetClient/README.md

```diff
@@ -57,6 +57,56 @@ the `ConfigureServices` of your `Startup` class. Refer to documentation for
 For an ASP.NET application, adding instrumentation is typically done in the
```

Comment:
I think we can Asp.Net section. as no .NET Framework implements netstandard2.1 right?

Reply:
Ah! Good 👁️.